python/paddle/dataset/sentiment.py

```diff
@@ -28,6 +28,9 @@
 
 import nltk
 from nltk.corpus import movie_reviews
+import ssl
+ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
Why should you add `ssl` here? It is not used in this PR, and if you add this line, we should `pip install ssl` before development. @sandyhouse

Reply:
add ssl to fix the issue#20749. I will launch a new pr to add ssl to requirements.txt.

Reply:
You should add `ssl` into `python/requirments.txt` for the usage.

Reply:
Right. I have opened the [pr#21669](https://github.com/PaddlePaddle/Paddle/pull/21669) to do that.